sources/packages/backend/libs/utilities/src/date-utils.ts

```diff
@@ -168,6 +168,24 @@ export const addDays = (daysToAdd: number, date?: Date | string): Date => {
     .toDate();
 };
 
+/**
+ * Add units amounts (days, months, years) to a date and returns it as a date only ISO string.
```

Comment:
not at all a blocker, just my thought, with all amounts (money) in our SIMS, when I first read it, I wondered what is the amount here. Then I realized 😄 the context here.
 can we use something like value or something else here?

Reply:
I see the point. I can change it to value.

Reply:
Changed!